manifests/base/argo-server/argo-server-deployment.yaml

```diff
@@ -16,6 +16,9 @@ spec:
         - name: argo-server
           image: argoproj/argocli:latest
           securityContext:
+            readOnlyRootFilesystem: true
```

Comment:
The server needs to write to /tmp for artefact download.

Reply:
Thanks, I do see the following present already, would that suffice @alexec ?
```
  volumeMounts:
            - mountPath: /tmp
              name: tmp
```

Reply:
Have you checked this works?